cmd/experimental/kjobctl/pkg/builder/builder.go

```diff
@@ -236,18 +247,36 @@ func (b *Builder) buildPodSpec(templateSpec corev1.PodSpec) corev1.PodSpec {
 
 		container.VolumeMounts = append(container.VolumeMounts, bundle.Spec.ContainerVolumeMounts...)
 		container.Env = append(container.Env, bundle.Spec.EnvVars...)
+		container.Env = append(container.Env, b.additionalEnvironmentVariables()...)
 	}
 
 	for i := range templateSpec.InitContainers {
 		initContainer := &templateSpec.InitContainers[i]
 
 		initContainer.VolumeMounts = append(initContainer.VolumeMounts, bundle.Spec.ContainerVolumeMounts...)
 		initContainer.Env = append(initContainer.Env, bundle.Spec.EnvVars...)
+		initContainer.Env = append(initContainer.Env, b.additionalEnvironmentVariables()...)
 	}
 
 	return templateSpec
 }
 
+func (b *Builder) additionalEnvironmentVariables() []corev1.EnvVar {
+	userID := os.Getenv(constants.SystemEnvVarNameUser)
+	timestamp := b.buildTime.Format(time.RFC3339)
+	taskName := fmt.Sprintf("%s_%s", b.namespace, b.profileName)
```

Comment:
@mwielgus @alculquicondor 
Now we are generating name on server side, using GenerateName with `profile-name-` prefix. Is it ok to use just namespace_profile? Or should we generate name in CLI side?

Reply:
It seems ok.